pkg/plugin/pr-sanitizer/pr_sanitizer_message_builder.go

```diff
@@ -0,0 +1,59 @@
+package prsanitizer
+
+import "strings"
+
+type failureMessageBuilder struct {
+	description []string
+}
+
+// FailureMessage is message containing failure reasons of pr-sanitizer.
+type FailureMessage string
+
+// FailureMessageBuilder is builder to build failure message for description.
+type FailureMessageBuilder interface {
+	Title(isValid bool) FailureMessageBuilder
+	Description(desc string) FailureMessageBuilder
+	IssueLink(isIssueLinked bool) FailureMessageBuilder
+	Build() FailureMessage
+}
+
+const (
+	// TitleFailure is a message used in GH Status as description when the PR title does not follow semantic message style
+	TitleFailure = "PR title does not conform with semantic commit message style."
+
+	// IssueLinkMissing is a message used in GH Status as description when no tests shipped with the PR
+	IssueLinkMissing = "Issue link is missing in this PR description."
+
+	// DescriptionLengthShort is a message used in GH Status as description when the PR description does not have minimum required characters.
+	DescriptionLengthShort = "PR description is too short, expecting more than 50 characters."
```

Comment:
These descriptions are great, I also like the API, however, the longer content should be part of the status message, not the status of the builds - the message can be too long and thus hard to read.